backend/tests/graphql/slackTeam/mutations/deleteSlackTeam.test.js

```diff
@@ -0,0 +1,63 @@
+import TestSuiteUtils from "../../../utils";
+
+const utils = new TestSuiteUtils();
+utils.setupJest();
+
+let adminContext = null;
```

Comment:
using closure vars like this seems ugly, but I have yet to find a better way. Ideas?